packages/mobile/src/screens/new-item/components/new-item-form/new-item-form.tsx

```diff
@@ -0,0 +1,207 @@
+  site?: string;
+};
+
+const NewItemForm: FC = () => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+
+  const { control, errors } = useAppForm<NewItem>({
+    defaultValues: {
+      currency: 'UAH',
+      country: 'Ukraine',
+      callingCode: 'UA',
+    },
+  });
+
+  return (
+    <View>
+      <Text style={[globalStyles.fs14, globalStyles.mt5, styles.title]}>
+        {t('make_a_post.DOWNLOAD_PHOTOS')}
+      </Text>
+      <AddPhotos />
+      <Text style={[globalStyles.fs14, globalStyles.mt5, styles.title]}>
+        {t('make_a_post.DESCRIPTION')}
+      </Text>
+      <Input
+        label={t('make_a_post.CATEGORY')}
+        placeholder={t('make_a_post.CATEGORY_PLACEHOLDER')}
+        name="category"
+        control={control}
+        errors={errors}
+        contentContainerStyle={globalStyles.mt5}
+      />
+      <Input
+        label={t('make_a_post.TITLE_NAME')}
+        placeholder={t('make_a_post.TITLE_NAME_PLACEHOLDER')}
+        name="titleName"
+        control={control}
+        errors={errors}
+        contentContainerStyle={globalStyles.mt5}
+      />
+      <Input
+        label={t('make_a_post.DESCRIPTION')}
+        placeholder={t('make_a_post.DESCRIPTION_PLACEHOLDER')}
+        name="description"
+        control={control}
+        errors={errors}
+        contentContainerStyle={globalStyles.mt5}
+      />
+
+      <View style={[styles.row, globalStyles.mt5]}>
+        <View style={styles.leftWrap}>
+          <Popover
+            popoverStyle={styles.popover}
+            from={
+              <TouchableOpacity style={styles.tooltipWrap}>
+                <Text style={[globalStyles.fs12]}>
+                  {t('make_a_post.PRICE')}
+                </Text>
+                <InfoIcon
+                  size={13}
+                  color={ColorPalette.YELLOW_200}
+                  style={styles.tooltipIcon}
+                />
+              </TouchableOpacity>
+            }
+          >
+            <Text>{t('make_a_post.PRICE_TOOLTIP')}</Text>
+          </Popover>
+        </View>
+        <View style={styles.rightWrap}>
+          <Text style={[globalStyles.fs12]}>{t('make_a_post.CURRENCY')}</Text>
+        </View>
+      </View>
+
+      <View style={styles.row}>
+        <Input
+          label=""
+          placeholder={t('make_a_post.PRICE_PLACEHOLDER')}
+          name="price"
+          control={control}
+          errors={errors}
+          contentContainerStyle={styles.leftInput}
+        />
+
+        <View style={styles.rightInput}>
+          <DropDown
+            name="currency"
+            control={control}
+            items={CURRENCY}
+            zIndex={25}
+          />
+        </View>
+      </View>
+
+      <Text style={[globalStyles.fs14, globalStyles.mt6, styles.title]}>
+        {t('make_a_post.CONTACT')}
+      </Text>
+      <DropDown
+        label={t('make_a_post.COUNTRY')}
+        name="country"
+        control={control}
+        items={COUNTRIES}
```

Comment:
For now it's only Ukraine, so we don't actually need dropdown here. Disabled input is seemed to be enough